### DISCUSSION TITLE
Idea_By_team_x

### DISCUSSION THREAD

**Discussion by vgudur** [msusdev/Innovation-Challenge#14] [Ideas]
Discussion for TeamX 

KOSSI DIEUDONNE EKPO | kossidieudonne.ekpo@outlook.fr
SANVEE AHLONKO | fabrice@ndv1.onmicrosoft.com
Maria | nintendoassirou@gmail.com
Tiana Aprianti | tianaaprianti85@gmail.com

**Comment by vgudur** [comment]


**Reply by francisca092** [reply]
Me, Togo - West Africa

**Reply by TiaApril** [reply]
Hi everyone, I'm Tiana, I'm based in MN, US

**Comment by vgudur** [comment]
Who is Togo? We want to make sure only our team is enabled to use
particular discussion board in github.
